app/src/main/res/values-tr/strings.xml

```diff
@@ -0,0 +1,49 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="app_version">Uygulama Versiyonu</string>
+    <string name="back">Geri</string>
+    <string name="bug_report">Hata Raporla</string>
+    <string name="cancel">Vazgeç</string>
+    <string name="classic_game">Klasik Oyun</string>
+    <string name="confirm_restart_accept">Yeniden Başlat</string>
+    <string name="confirm_restart_message">Oyun yeniden başlatılsın mı?</string>
+    <string name="confirm_restart_title">Yeni Oyun</string>
+    <string name="game_won">Kazandın!</string>
+    <string name="greet">Hoşgeldin</string>
+    <string name="guide">Eğitici</string>
+    <string name="got_it">Anladım!</string>
+    <string name="combos">Kombolar</string>
+    <string name="achievements">Başarılar</string>
+    <string name="toast_login_failed">Giriş yapılamadı</string>
+    <string name="toast_no_possibility">Hamle kalmadı</string>
+    <string name="statistics">İstatistikler</string>
+    <string name="sign_out">Çıkış yap</string>
+    <string name="sign_in">Giriş yap</string>
+    <string name="signed_in">Giriş yapıldı</string>
+    <string name="signed_in_as">Olarak giriş yapıldı :</string>
+    <string name="toast_exit">Çıkmak için tekrar basın</string>
+    <string name="menu">Menü</string>
+    <string name="random_game">Rastgele Oyun</string>
+    <string name="rate">Oyla</string>
+    <string name="rate_cancel">Hayır, Teşekkürler</string>
+    <string name="rate_later">Sonra</string>
+    <string name="rate_message">Eğer #-ludus'u sevdiyseniz, bir dakikanızı ayırıp oylayın!</string>
```

Comment:
[Travis-ci](https://travis-ci.org/FallenRiteMonk/ludus/builds/81379248) throws a error on this line due to the apostrophe. Have a look at [this link](http://stackoverflow.com/questions/15705647/apostrophe-not-preceded-by) to fix this!